src/main/kotlin/org/pkl/lsp/Stdlib.kt

```diff
@@ -1,80 +0,0 @@
-/**
- * Copyright © 2024 Apple Inc. and the Pkl project authors. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     https://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.pkl.lsp
-
-import java.io.IOException
-import java.net.URI
-import org.pkl.core.parser.Parser
-import org.pkl.core.util.IoUtils
-import org.pkl.lsp.ast.PklModule
-import org.pkl.lsp.ast.PklModuleImpl
-
-class Stdlib(project: Project) : Component(project) {
```

Comment:
I removed this because it's not really needed anymore, because every file is tracked by `VirtualFileManager` via its URI.

So, getting the base module is simply `virtualFileManager.get(URI("pkl:base")).get!!`, and that will return the same module every time.

This also means that we no longer need to initialize the standard library upon startup.